test/Microsoft.AspNetCore.Mvc.TestCommon/Microsoft.AspNetCore.Mvc.TestCommon.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I think we might be able to add a comment to the .notest file above saying that this is a "feature" of KoreBuild. Don't think it looks at the content.